editor/plugins/gradient_texture_2d_editor_plugin.cpp

```diff
@@ -180,6 +180,7 @@ GradientTexture2DEditorRect::GradientTexture2DEditorRect() {
 
 	checkerboard = memnew(TextureRect);
 	checkerboard->set_stretch_mode(TextureRect::STRETCH_TILE);
+	checkerboard->set_ignore_texture_size(true);
 	checkerboard->set_draw_behind_parent(true);
 	add_child(checkerboard);
```

Comment:
In fact, the `checkerboard` should probably be added as an internal node to make potential breaking things a little harder.
```suggestion
	add_child(checkerboard, false, INTERNAL_MODE_FRONT);
```